docs/stable/raw/aws-sno.csv

```diff
@@ -10,7 +10,6 @@ Ingress,TCP,6443,openshift-kube-apiserver,apiserver,kube-apiserver,kube-apiserve
 Ingress,TCP,8080,openshift-network-operator,,network-operator,network-operator,master,FALSE
 Ingress,TCP,8798,openshift-machine-config-operator,machine-config-daemon,machine-config-daemon,machine-config-daemon,master,FALSE
 Ingress,TCP,9001,openshift-machine-config-operator,machine-config-daemon,machine-config-daemon,kube-rbac-proxy,master,FALSE
-Ingress,TCP,9099,openshift-cluster-version,cluster-version-operator,cluster-version-operator,cluster-version-operator,master,FALSE

```

Comment:
What is this port? I don't see it in the ignore list

Reply:
This is a port that are ignoring from the function that i created 
The reason that i created that function to dont have so much list of ports on it
Since we know about a filter that we are sure about it
Its the filter of the ports that have service and that service are internal and the pods have host network